src/engine/authentication.md

```diff
@@ -51,4 +49,60 @@ Other claims **MAY** be included in the JWT payload. If the execution layer clie
 
 ## Examples
 
-Todo, add some examples of JWT authentication here.
```

Comment:
Sorry I don't think this is really what we expected in terms of examples. I was thinking something more like https://jwt.io which shows the message, the jwt signature, and the verification of the signature.

Reply:
This code solves the TODO by providing an example. If you don't want people spending time helping, then don't write vague TODOs.